<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configWindow</class>
 <widget class="QWidget" name="configWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget{ 
	color: rgb(104, 112, 130); 
	font-size:14px; 
	border-style:solid; 
	border-width:1px; 
	border-color:rgb(229, 231, 234); 
	border-radius:4px; 
} 
QTableWidget::item{ 
	border-style:solid; 
	border-bottom-width:1px; 
	border-color:rgb(229, 231, 234); 
} 
QPushButton:hover{
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
	padding-left:15px;
	padding-right:15px;
	background-color: rgba(47, 105, 235, 0.08);
}
QPushButton {
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
	padding-left:15px;
	padding-right:15px;
}
QPushButton:pressed{
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
	padding-left:15px;
	padding-right:15px;
	background-color: rgba(47, 105, 235, 0.2);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_config" native="true">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section{
color: rgb(104, 112, 130);
font-size:14px;
background: transparent;
background: rgb(246, 249, 255);
padding: 1px;border: none;
border-style:solid;
border-right-width:0px;
border-left-width:0px;
}
QTableWidget{
color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-color:rgb(229, 231, 234);
border-radius:4px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="search_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-height：20px;
border-color:rgb(229, 231, 234);
border-radius:25px;
padding-left:5px;
</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="maxLength">
           <number>6</number>
          </property>
          <property name="placeholderText">
           <string>仅英文</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="search_pushButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_configAdd">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="toolTip">
           <string>添加产品信息</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(47, 105, 235);
font-size:14px;
border-style:solid;
border-width:1px;
border-color:rgb(47, 105, 235);
border-radius:4px;
padding-left:15px;
padding-right:15px;</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_configDel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="toolTip">
           <string>删除产品信息</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(47, 105, 235);
font-size:14px;
border-style:solid;
border-width:1px;
border-color:rgb(47, 105, 235);
border-radius:4px;
padding-left:15px;
padding-right:15px;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_config">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableWidget::pane{
	border: 2px solid rgb(210, 210, 210);
	background:rgb(246, 246, 246);
	border-top-color:transparent;
}
QTableWidget::tab-bar{
	background:rgb(0, 0, 0);
	subcontrol-position:left;
}
QTableBar::tab{
	width:173px;
	height:45px;
	background:rgb(210, 210, 210);
	border: 2px solid rgb(210, 210, 210);
	border-top-left-radius: 8px;
	border-top-right-radius: 8px;
}
QTableBar::tab:selected{	
	background:rgb(246, 246, 246);
	border-bottom-color:rgb(246, 246, 246);
}
QTableBar::tab:!selected{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(240, 240, 240, 255), stop:0.5 rgba(210, 210, 210, 255), stop:1 rgba(225, 225, 225, 255));
}
</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>产品号</string>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>产品密钥</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数据格式</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>备注</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_info">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string>搜索内容不能为空</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
